pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -175,13 +175,18 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 	// DELETE: the CR was deleted
 	if !cephFilesystemSubVolumeGroup.GetDeletionTimestamp().IsZero() {
 		logger.Debugf("deleting subvolume group %q", cephFilesystemSubVolumeGroup.Name)
-		err := r.deleteSubVolumeGroup(cephFilesystemSubVolumeGroup)
-		if err != nil {
-			if strings.Contains(err.Error(), opcontroller.UninitializedCephConfigError) {
-				logger.Info(opcontroller.OperatorNotInitializedMessage)
-				return opcontroller.WaitForRequeueIfOperatorNotInitialized, nil
+		// On external cluster, we don't delete the subvolume group, it has to be deleted manually
+		if cephCluster.Spec.External.Enable {
+			logger.Warning("external subvolume group deletion is not supported, delete it manually")
+		} else {
+			err := r.deleteSubVolumeGroup(cephFilesystemSubVolumeGroup)
+			if err != nil {
+				if strings.Contains(err.Error(), opcontroller.UninitializedCephConfigError) {
+					logger.Info(opcontroller.OperatorNotInitializedMessage)
+					return opcontroller.WaitForRequeueIfOperatorNotInitialized, nil
+				}
+				return reconcile.Result{}, errors.Wrapf(err, "failed to delete ceph ceph filesystem subvolume group %q", cephFilesystemSubVolumeGroup.Name)
```

Comment:
`failed to delete ceph ceph filesystem subvolume group` to `failed to delete ceph filesystem subvolume group`